reconcile/acs_policies.py

```diff
@@ -3,6 +3,7 @@
 from typing import cast
 
 import reconcile.gql_definitions.acs.acs_policies as gql_acs_policies
+from reconcile.acs_notifiers import AcsNotifiersIntegration
```

Comment:
TBH, I don't like it if one integration uses another integration. This is always hard to refactor. Do you mind and move the needed parts into a utils file? Especially, `AcsNotifiersIntegration._build_jira_notifier`, as a "private" method :(